internal/pkg/githubactions/githubactions.go

```diff
@@ -109,27 +109,28 @@ func (ga *GithubActions) DeleteWorkflow(workflow *Workflow) error {
 	return nil
 }
 
-func (ga *GithubActions) fileExists(filename string) (bool, error) {
-	_, _, resp, err := ga.client.Repositories.GetContents(
+// getFileSHA: if file exists, return SHA, or return "", for file check and file delete
```

Comment:
Maybe this comment could be more clear:
1. file exists and no error occurred: return (string(SHA), nil)
2. some errors occurred: return ("", errl)
3. file not exists and no error occurred: return ("", nil)

Reply:
function note added